messagequeues/Receiver/Worker.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.IO;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Tesseract;
+using System.Drawing;
+
+namespace Receiver;
+
+public class Worker : BackgroundService
+{
+    private readonly ILogger<Worker> _logger;
+
+    public Worker(ILogger<Worker> logger)
+    {
+        _logger = logger;
+    }
+
+    protected override async Task ExecuteAsync(CancellationToken stoppingToken)

```

Comment:
make sure, that "async" shouldn't be used if it's not required https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/async-scenarios#important-info-and-advice

Reply:
You are right its coming when I create service default method :) I will change and committed

Reply:
https://learn.microsoft.com/en-us/dotnet/core/extensions/workers its coming here this method

Reply:
If there is no `await` used in the method body, you need to delete `async` keyword and return completed task